packages/components/src/components/date-picker/date-picker-native.tsx

```diff
@@ -4,6 +4,23 @@ import { toMoment } from '@deriv/shared';
 import Icon from '../icon';
 import Text from '../text';
 
+type TDatePickerNativeProps = Omit<React.HTMLAttributes<HTMLInputElement>, 'onSelect'> & {
+    value: string;
```

Comment:
```suggestion
    value: string | null;
```

Reply:
<img width="860" alt="image" src="https://user-images.githubusercontent.com/129206554/230573600-c70b11e7-02fb-49b0-a16c-5fc5cf293a96.png">

Reply:
<img width="696" alt="image" src="https://user-images.githubusercontent.com/129206554/230574208-556f86d0-fbf1-48ab-86b0-60201f861f9b.png">

adding this null check works fine @farzin-deriv 

I didn't put `(input_ref.current && value) ...` because it won't allow the input value to get empty

Reply:
I thought of this code again and find an issue with the above line, `value !== null` will filter out the `null` value from prop and results in performing no action if the value is `null`

instead I did this : `if (input_ref.current) input_ref.current.value = value || '';` this way it will behave like if `null` is passed then make the input empty, which I think is expected